src/screens/ProfileScreen/elements/BottomSheetBio.js

```diff
@@ -77,7 +82,7 @@ const styles = StyleSheet.create({
   textStyling: {
     fontFamily: fonts.inter[600],
     fontSize: 18,
-    color: COLORS.white
+    color: COLORS.white2
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the text styling has been changed from `COLORS.white` to `COLORS.white2`. This change should provide better consistency with the overall color scheme of the application. However, ensure that `COLORS.white2` provides enough contrast against its background for accessibility purposes.
```diff
- color: COLORS.white
+ color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->